--- wireless_tools.28/iwlib.h.orig	2005-12-06 02:18:06.000000000 +0100
+++ wireless_tools.28/iwlib.h	2006-03-20 00:03:54.000000000 +0100
@@ -32,12 +32,6 @@
 #include <sys/socket.h>		/* For AF_INET & struct sockaddr */
 #include <netinet/in.h>         /* For struct sockaddr_in */
 #include <netinet/if_ether.h>
-
-/* Fixup to be able to include kernel includes in userspace.
- * Basically, kill the sparse annotations... Jean II */
-#ifndef __user
-#define __user
-#endif
 
 #include <linux/types.h>		/* for "caddr_t" et al		*/
 
--- wireless_tools.28/wireless.19.h.orig	2006-03-20 00:02:13.000000000 +0100
+++ wireless_tools.28/wireless.19.h	2006-03-20 00:04:46.000000000 +0100
@@ -641,7 +641,7 @@
  */
 struct	iw_point
 {
-  void __user	*pointer;	/* Pointer to the data  (in user space) */
+  void		*pointer;	/* Pointer to the data  (in user space) */
   __u16		length;		/* number of fields or size in bytes */
   __u16		flags;		/* Optional params */
 };
--- wireless_tools.28/wireless.20.h.orig	2006-05-27 18:51:20.000000000 +0200
+++ wireless_tools.28/wireless.20.h	2006-05-27 18:51:29.000000000 +0200
@@ -645,7 +645,7 @@
  */
 struct	iw_point
 {
-  void __user	*pointer;	/* Pointer to the data  (in user space) */
+  void		*pointer;	/* Pointer to the data  (in user space) */
   __u16		length;		/* number of fields or size in bytes */
   __u16		flags;		/* Optional params */
 };
--- wireless_tools.28/wireless.18.h.orig	2006-05-27 18:50:55.000000000 +0200
+++ wireless_tools.28/wireless.18.h	2006-05-27 18:51:08.000000000 +0200
@@ -626,7 +626,7 @@
  */
 struct	iw_point
 {
-  void __user	*pointer;	/* Pointer to the data  (in user space) */
+  void		*pointer;	/* Pointer to the data  (in user space) */
   __u16		length;		/* number of fields or size in bytes */
   __u16		flags;		/* Optional params */
 };
--- wireless_tools.29/wireless.21.h.orig	2006-03-25 01:26:36.000000000 +0100
+++ wireless_tools.29/wireless.21.h	2006-10-02 16:37:49.583457000 +0200
@@ -680,7 +680,7 @@
  */
 struct	iw_point
 {
-  void __user	*pointer;	/* Pointer to the data  (in user space) */
+  void		*pointer;	/* Pointer to the data  (in user space) */
   __u16		length;		/* number of fields or size in bytes */
   __u16		flags;		/* Optional params */
 };
